docs/native-modules-intro.md

```diff
@@ -13,10 +13,11 @@ The NativeModule system exposes instances of Java/Objective-C/C++ (native) class
 
 ## Native Module Setup
 
-There are two ways to write a native module for your React Native application:
+There are three ways to write a native module for your React Native application:
```

Comment:
```suggestion
There are different ways to write a native module for your React Native application:
```

so if in the future we add/remove one of the way, we don't have to change this line anymore.